core/domain/exp_domain.py

```diff
@@ -1611,6 +1611,57 @@ def create_default_state(
                 default_dest_state_name))
 
 
+class ExplorationVersionsDiff(object):
+    """Domain object for the difference between two versions of an Oppia
+    exploration.
+
+    Attributes:
+        states_added: list(str). The states added to the exploration from
```

Comment:
added_state_names, deleted_state_names, new_to_old_state_names

(they're state names, not states -- might want to change the docstrings accordingly e.g. "Names of states added ...")

Reply:
Done.